---
title: "Phylogenomics, phenotypic, and functional traits of five novel (Earth-derived) bacterial species isolated from the International Space Station and their prevalence in metagenomes"
description: "Abstract With the advent of long-term human habitation in space and on the moon, understanding how the built environment microbiome of space habitats differs from Earth habitats, and how microbes surv..."
authors: "Anna C Simpson; Pratyay Sengupta; Flora Zhang; Asif Hameed; Ceth W Parker; Nitin K Singh; Georgios Miliotis; Punchappady D Rekha; Karthik Raman; Christopher E Mason; Kasthuri Venkateswaran"
url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10628120/
---

With the advent of long-term human habitation in space and on the moon, understanding how the built environment microbiome of space habitats differs from Earth habitats, and how microbes survive, proliferate and spread in space conditions, is becoming more important. The microbial tracking mission series has been monitoring the microbiome of the International Space Station (ISS) for almost a decade. During this mission series, six unique strains of Gram-stain-positive bacteria, including two spore-forming and three non-spore-forming species, were isolated from the environmental surfaces of the ISS. The analysis of their 16S rRNA gene sequences revealed > 99% similarities with previously described bacterial species. To further explore their phylogenetic affiliation, whole genome sequencing was undertaken. For all strains, the gyrB gene exhibited < 93% similarity with closely related species, which proved effective in categorizing these ISS strains as novel species. Average nucleotide identity and digital DNA–DNA hybridization values, when compared to any known bacterial species, were < 94% and <50% respectively for all species described here. Traditional biochemical tests, fatty acid profiling, polar lipid, and cell wall composition analyses were performed to generate phenotypic characterization of these ISS strains. A study of the shotgun metagenomic reads from the ISS samples, from which the novel species were isolated, showed that only 0.1% of the total reads mapped to the novel species, supporting the idea that these novel species are rare in the ISS environments. In-depth annotation of the genomes unveiled a variety of genes linked to amino acid and derivative synthesis, carbohydrate metabolism, cofactors, vitamins, prosthetic groups, pigments, and protein metabolism. Further analysis of these ISS-isolated organisms revealed that, on average, they contain 46 genes associated with virulence, disease, and defense. The main predicted functions of these genes are: conferring resistance to antibiotics and toxic compounds, and enabling invasion and intracellular resistance. After conducting antiSMASH analysis, it was found that there are roughly 16 cluster types across the six strains, including β-lactone and type III polyketide synthase (T3PKS) clusters. Based on these multi-faceted taxonomic methods, it was concluded that these six ISS strains represent five novel species, which we propose to name as follows: Arthrobacter burdickii IIF3SC-B10 T (= NRRL B-65660 T = DSM 115933 T ), Leifsonia virtsii F6_8S_P_1A T (= NRRL B-65661 T = DSM 115931 T ), Leifsonia williamsii F6_8S_P_1B T (= NRRL B-65662 T = DSM 115932 T ), Paenibacillus vandeheii F6_3S_P_1C T (= NRRL B-65663 T = DSM 115940 T ), and Sporosarcina highlanderae F6_3S_P_2 T (= NRRL B-65664 T = DSM 115943 T ). Identifying and characterizing the genomes and phenotypes of novel microbes found in space habitats, like those explored in this study, is integral for expanding our genomic databases of space-relevant microbes. This approach offers the only reliable method to determine species composition, track microbial dispersion, and anticipate potential threats to human health from monitoring microbes on the surfaces and equipment within space habitats. By unraveling these microbial mysteries, we take a crucial step towards ensuring the safety and success of future space missions. Subject terms: Environmental microbiology, Functional genomics, Genome, Prokaryote
