---
title: "Microbial Tracking-2, a metagenomics analysis of bacteria and fungi onboard the International Space Station"
description: "Abstract Background The International Space Station (ISS) is a unique and complex built environment with the ISS surface microbiome originating from crew and cargo or from life support recirculation i..."
authors: "Camilla Urbaniak; Michael D Morrison; James B Thissen; Fathi Karouia; David J Smith; Satish Mehta; Crystal Jaing; Kasthuri Venkateswaran"
url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9241228/
---

Background The International Space Station (ISS) is a unique and complex built environment with the ISS surface microbiome originating from crew and cargo or from life support recirculation in an almost entirely closed system. The Microbial Tracking 1 (MT-1) project was the first ISS environmental surface study to report on the metagenome profiles without using whole-genome amplification. The study surveyed the microbial communities from eight surfaces over a 14-month period. The Microbial Tracking 2 (MT-2) project aimed to continue the work of MT-1, sampling an additional four flights from the same locations, over another 14 months. Methods Eight surfaces across the ISS were sampled with sterile wipes and processed upon return to Earth. DNA extracted from the processed samples (and controls) were treated with propidium monoazide (PMA) to detect intact/viable cells or left untreated and to detect the total DNA population (free DNA/compromised cells/intact cells/viable cells). DNA extracted from PMA-treated and untreated samples were analyzed using shotgun metagenomics. Samples were cultured for bacteria and fungi to supplement the above results. Results Staphylococcus sp. and Malassezia sp. were the most represented bacterial and fungal species, respectively, on the ISS. Overall, the ISS surface microbiome was dominated by organisms associated with the human skin. Multi-dimensional scaling and differential abundance analysis showed significant temporal changes in the microbial population but no spatial differences. The ISS antimicrobial resistance gene profiles were however more stable over time, with no differences over the 5-year span of the MT-1 and MT-2 studies. Twenty-nine antimicrobial resistance genes were detected across all samples, with macrolide/lincosamide/streptogramin resistance being the most widespread. Metagenomic assembled genomes were reconstructed from the dataset, resulting in 82 MAGs. Functional assessment of the collective MAGs showed a propensity for amino acid utilization over carbohydrate metabolism. Co-occurrence analyses showed strong associations between bacterial and fungal genera. Culture analysis showed the microbial load to be on average 3.0 Ã— 10 5 cfu/m 2 Conclusions Utilizing various metagenomics analyses and culture methods, we provided a comprehensive analysis of the ISS surface microbiome, showing microbial burden, bacterial and fungal species prevalence, changes in the microbiome, and resistome over time and space, as well as the functional capabilities and microbial interactions of this unique built microbiome. Data from this study may help to inform policies for future space missions to ensure an ISS surface microbiome that promotes astronaut health and spacecraft integrity. Your browser is not supporting the HTML5 <video> element. You may download that video as a file and play it with a player of you choice. Download video stream . Download video file (99.8MB, mp4) Open in a new tab Video Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s40168-022-01293-0. Keywords: International Space Station, Microbial monitoring, Microbiome, Metagenomics, Microbial tracking, Built environment
