---
title: "Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces"
description: "Abstract Background The International Space Station (ISS) is a closed system inhabited by microorganisms originating from life support systems, cargo, and crew that are exposed to unique selective pre..."
authors: "Aleksandra Checinska Sielaff; Camilla Urbaniak; Ganesh Babu Malli Mohan; Victor G Stepanov; Quyen Tran; Jason M Wood; Jeremiah Minich; Daniel McDonald; Teresa Mayer; Rob Knight; Fathi Karouia; George E Fox; Kasthuri Venkateswaran"
url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6452512/
---

Background The International Space Station (ISS) is a closed system inhabited by microorganisms originating from life support systems, cargo, and crew that are exposed to unique selective pressures such as microgravity. To date, mandatory microbial monitoring and observational studies of spacecraft and space stations have been conducted by traditional culture methods, although it is known that many microbes cannot be cultured with standard techniques. To fully appreciate the true number and diversity of microbes that survive in the ISS, molecular and culture-based methods were used to assess microbial communities on ISS surfaces. Samples were taken at eight pre-defined locations during three flight missions spanning 14Â months and analyzed upon return to Earth. Results The cultivable bacterial and fungal population ranged from 10 4 to 10 9 CFU/m 2 depending on location and consisted of various bacterial ( Actinobacteria , Firmicutes , and Proteobacteria ) and fungal ( Ascomycota and Basidiomycota ) phyla. Amplicon sequencing detected more bacterial phyla when compared to the culture-based analyses, but both methods identified similar numbers of fungal phyla. Changes in bacterial and fungal load (by culture and qPCR) were observed over time but not across locations. Bacterial community composition changed over time, but not across locations, while fungal community remained the same between samplings and locations. There were no significant differences in community composition and richness after propidium monoazide sample treatment, suggesting that the analyzed DNA was extracted from intact/viable organisms. Moreover, approximately 46% of intact/viable bacteria and 40% of intact/viable fungi could be cultured. Conclusions The results reveal a diverse population of bacteria and fungi on ISS environmental surfaces that changed over time but remained similar between locations. The dominant organisms are associated with the human microbiome and may include opportunistic pathogens. This study provides the first comprehensive catalog of both total and intact/viable bacteria and fungi found on surfaces in closed space systems and can be used to help develop safety measures that meet NASA requirements for deep space human habitation. The results of this study can have significant impact on our understanding of other confined built environments on the Earth such as clean rooms used in the pharmaceutical and medical industries. Electronic supplementary material The online version of this article (10.1186/s40168-019-0666-x) contains supplementary material, which is available to authorized users. Keywords: International Space Station, Microbiome, 16S rRNA, ITS, Environmental surface, Built microbiome, Propidium monoazide, Microbial diversity
